### DISCUSSION TITLE
6 Months trying and I cannot make the connection work, please help!

### DISCUSSION THREAD

**Discussion by taimurt** [ParadoxAlarmInterface/pai#436] [Q&A] [answered]
Hello,

I have Paradox sp6000 and I tried to connect using different ESP32 versions as well ESP8266 with HomeAssitant but the connection always timeouts. It has been 6 months I have been trying and now finally I am asking you guys for help.

What is working:
-Local Home Assitant instance with PAI installed and working MQTT connection.

What I have tried:
-Tried all passwords
-When nothing worked, I reset the whole Paradox sp6000 to factory settings and then I tried the password 0000
-Tired 1234, 123456, paradox, 0000, 00000 passwords.
-I used ESP32 Devkit v1 Doit version as well as M5stack and ESP8266
-I switched the pins for RX and TX for each password. 
-I have waited multiple days to try again and tried arming and disarming the system.

Little history:
The system was installed by the previous owners of the house in probably 2011 or 2012. They first had some radio frequency devices that were connected to the security company but later in 2020, they installed the LTE module with a serial connection. When I purchased the house, they just removed the SIM card from the module and gave me the arm and disarm codes. I have not tried to use the LTE module myself.

See the attached picture of the SP6000. I am also unsure if I buy the ip150 module, if it will work or not due to different versions not being compatible with this PAI home assistant addon.

![4](https://github.com/ParadoxAlarmInterface/pai/assets/4950690/d2573cdb-cb9f-41b2-8df7-ddb93dd7a479)

Following is my PAI config as well as the connection logs.

<img width="804" alt="Screenshot 2024-02-11 at 21 00 16" src="https://github.com/ParadoxAlarmInterface/pai/assets/4950690/bf9af900-8468-4847-881c-730ecb109cb4">

LOGGING_LEVEL_CONSOLE: 20
LOGGING_LEVEL_FILE: 40
CONNECTION_TYPE: IP
SERIAL_PORT: /dev/ttyUSB0
SERIAL_BAUD: 9600
IP_CONNECTION_HOST: 192.168.0.45
IP_CONNECTION_PORT: 10000
IP_CONNECTION_PASSWORD: aaaa
KEEP_ALIVE_INTERVAL: 10
LIMITS:
  zone: auto
  user: 1-10
  door: ""
  pgm: 1-5
  partition: auto
  module: ""
  repeater: ""
  keypad: ""
  key-switch: ""
SYNC_TIME: true
SYNC_TIME_MIN_DRIFT: 120
PASSWORD: "0000"
MQTT_ENABLE: true
MQTT_HOST: core-mosquitto
MQTT_PORT: 1883
MQTT_KEEPALIVE: 60
MQTT_USERNAME: mqtt656
MQTT_PASSWORD: xxxxx
MQTT_HOMEASSISTANT_AUTODISCOVERY_ENABLE: true
COMMAND_ALIAS:
  arm: partition all arm
  disarm: partition all disarm
MQTT_COMMAND_ALIAS:
  armed_home: arm_stay
  armed_night: arm_sleep
  armed_away: arm
  disarmed: disarm
HOMEASSISTANT_NOTIFICATIONS_EVENT_FILTERS:
  - live,alarm,-restore
  - trouble,-clock
  - live,tamper
HOMEASSISTANT_PUBLISH_PARTITION_PROPERTIES:
  - target_state
  - current_state
HOMEASSISTANT_PUBLISH_ZONE_PROPERTIES:
  - open
  - tamper
PUSHBULLET_CONTACTS: []
PUSHBULLET_EVENT_FILTERS:
  - live,alarm,-restore
  - trouble,-clock
  - live,tamper
PUSHOVER_EVENT_FILTERS:
  - live,alarm,-restore
  - trouble,-clock
  - live,tamper
PUSHOVER_BROADCAST_KEYS: []
SIGNAL_CONTACTS: []
SIGNAL_EVENT_FILTERS:
  - live,alarm,-restore
  - trouble,-clock
  - live,tamper
GSM_CONTACTS: []
GSM_EVENT_FILTERS:
  - live,alarm,-restore
  - trouble,-clock
  - live,tamper
IP_INTERFACE_ENABLE: false
IP_INTERFACE_PASSWORD: paradox
DUMMY_EVENT_FILTERS: []

**Comment by yozik04** [comment, marked as answer]
https://github.com/ParadoxAlarmInterface/pai/wiki/Connection-methods#serial-over-ip-esp32

I do not see these settings in your configuration.
```
IP_CONNECTION_BARE: true                      # As you are not connecting to IP150, then this is critical one.
IP_CONNECTION_PASSWORD: 'paradox' # Keep it 'paradox'. This is factory default.
PASSWORD: '0000'
```

PASSWORD is always 4 digits. Try `None`, `'0000'`, `'aaaa'` if it is factory reset.

**Reply by taimurt** [reply]
Wow, it is interesting that the "IP_CONNECTION_BARE: true" setting was not present in my configuration, which is why I could not select it. I had to add it manually in the YAML file, and then it appeared. It solved the problem, and now I have a connection. Thank you so much. 

<img width="906" alt="Screenshot 2024-02-12 at 00 24 53" src="https://github.com/ParadoxAlarmInterface/pai/assets/4950690/162436fa-6b03-46e5-a920-e8eab89052d7">

<img width="837" alt="Screenshot 2024-02-12 at 00 28 35" src="https://github.com/ParadoxAlarmInterface/pai/assets/4950690/436131cd-186c-4b2f-8cc6-ce6b6a7fa38a">

**Reply by yozik04** [reply]
Log tells that it fails to finish the connection process. But connection first step is done.
